helm/exim/templates/statefulsetOrdeployment.yaml

```diff
@@ -45,6 +58,26 @@ spec:
           - mountPath: "/var/spool/exim"
             name: {{ template "exim.name" . }}-data
           {{- end }}
+          {{- if .Values.global.enableTls }}
+          - mountPath: "/etc/ssl/default-out"
+            name: mailrelay-dev-ssl
+            readOnly: true
+          {{- end }}
+          {{- if .Values.global.enableInboundTls }}
+          - mountPath: "/etc/ssl/internal"
+            name: dev-inbound-mailrelay-tls
+            readOnly: true
+          {{- end }}
+          {{- if .Values.global.enableAuth }}
+          - mountPath: "/etc/exim/auth"
+            name: exim-passwd
+            readOnly: true
+          {{- end }}
+          {{- if .Values.global.enableAuth }}
```

Comment:
WIP: Mouth point. Would update this to be `default-out` in another commit. As i want to see how the output csi gives would be so i can decide on next steps